tests/usecases/data-processing-cloudant/manifest.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: data-processing-cloudant
+  version: 1.0
+  license: Apache-2.0
+  binding-package: /whisk.system/cloudant
```

Comment:
I assume this means the entire package is bound to this name.  This would prevent us from adding custom entities to this package